pages/doc/wavefront_spring_boot.md

```diff
@@ -459,13 +391,13 @@ You can invite users and let them send data to the same cluster:
   ```
   wavefront.freemium-account=true
   ```
-    {% include tip.html content="If you don’t want Wavefront to auto-negotiation a freemium account for you, set the value to `false`."%}
+    {% include tip.html content="If you don’t want Operations for Applications to auto-negotiation a freemium account for you, set the value to `false`."%}
 1. Restart your application.
 
 <a name="proxy"></a>
 <p><span style="font-size: large; font-weight: 500">Use the Wavefront Proxy</span></p>
 
-The [Wavefront proxy](proxies.html) ingests data and forwards them to the Wavefront service in a secure, fast, and reliable manner. It prevents data loss, simplifies firewall configuration, and allows you to filter or enrich data before they arrive at the Wavefront service.
+The [Wavefront proxy](proxies.html) ingests data and forwards them to our service in a secure, fast, and reliable manner. It prevents data loss, simplifies firewall configuration, and allows you to filter or enrich data before it is sent to our service.
```

Comment:
I'd use singular for data throughout:

...ingests data and forwards it to our service in a secure...